### PR TITLE
chore: Update README.md

### PR DESCRIPTION
**Pull request by gxjansen**
I've updated the layout of the badges. The second and third badge measure something differently but have the exact same text, making them less useful.

I didn't see an option to change the text inside the button, but by putting them in a table with headers, we now have some context as to what these buttons actually mean.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Oryx
 
-[![CI checks](https://github.com/spryker/oryx/actions/workflows/test.yml/badge.svg)](https://github.com/spryker/oryx/actions/workflows/test.yml)
-[![Oryx component dev env status](https://api.netlify.com/api/v1/badges/983ba7e1-0787-4036-a333-b1e01ef10fea/deploy-status)](https://app.netlify.com/sites/dev-oryx-components/deploys)
-[![Composable storefront dev env status](https://api.netlify.com/api/v1/badges/873ecabe-76b0-4493-af27-a0d150844cae/deploy-status)](https://app.netlify.com/sites/dev-composable-storefront/deploys)
+CI Checks | Oryx component dev env status | Composable storefront dev env status
+:-----: | :---: | :---:
+[![CI checks](https://github.com/spryker/oryx/actions/workflows/test.yml/badge.svg)](https://github.com/spryker/oryx/actions/workflows/test.yml) | [![Oryx component dev env status](https://api.netlify.com/api/v1/badges/983ba7e1-0787-4036-a333-b1e01ef10fea/deploy-status)](https://app.netlify.com/sites/dev-oryx-components/deploys) | [![Composable storefront dev env status](https://api.netlify.com/api/v1/badges/873ecabe-76b0-4493-af27-a0d150844cae/deploy-status)](https://app.netlify.com/sites/dev-composable-storefront/deploys)
```
Comment (PhilinTv) [inline README.md]:
```suggestion
[![CI checks](https://github.com/oryx-frontend/oryx/actions/workflows/test.yml/badge.svg)](https://github.com/oryx-frontend/oryx/actions/workflows/test.yml) | [![Oryx component dev env status](https://api.netlify.com/api/v1/badges/983ba7e1-0787-4036-a333-b1e01ef10fea/deploy-status)](https://app.netlify.com/sites/dev-oryx-components/deploys) | [![Composable storefront dev env status](https://api.netlify.com/api/v1/badges/873ecabe-76b0-4493-af27-a0d150844cae/deploy-status)](https://app.netlify.com/sites/dev-composable-storefront/deploys)
```

Comment (PhilinTv) [reply README.md]: We will fix netlify later as well.
@tolerants could you look at Netlify to resetup it with Oryx Frontend, even using the paid account for now? :)

**Review by gxjansen — COMMENTED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (tolerants) [reply README.md]: @PhilinTv sure.

Comment (tolerants) [reply README.md]: @PhilinTv Done. I've switched storybook and dev build for storefront for now. Later we will need production build and probably labs.

**Review by PhilinTv — APPROVED** [review]

**Pull request merged**